aws-athena/views/default-vw_pin_condo_char.sql

```diff
@@ -60,7 +60,9 @@ chars AS (
     SELECT
         par.parid AS pin,
         CASE
```

Comment:
**issue (blocking):** For anything like this, where there's a `CASE WHEN` without an `ELSE`, we should add a dbt test for not null.